common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1846,6 +1846,34 @@
     },
     "sqlState" : "42000"
   },
+  "INVALID_INVERSE_DISTRIBUTION_FUNCTION" : {
+    "message" : [
+      "Invalid inverse distribution function <funcName>."
```

Comment:
is `inverse distribution` a common terminology?

Reply:
I references H2 and Postgres, they call these function inverse distribution function.